src/shared/components/search-result/search-result.component.html

```diff
@@ -12,7 +12,7 @@ <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ caseSta
 
     <div *ngIf="(hasResults() || hasDrafts())" class="pagination-top"
       attr.aria-label="{{ getTotalResults() }} results have been found" role="status">
-      <span class="text-16" id="search-result-summary__text">Showing 
+      <span class="text-16" id="search-result-summary__text">Showing
```

Comment:
Might be a problem that the trailing whitespace has been stripped here - could this mess up the layout, or will the fact that the count is in a span mean that it won't be right next to the Showing text - please check

Reply:
@andywilkinshmcts - I have reverted this change and the space is now back. Thank you.